R/strategies.R

```diff
@@ -55,29 +66,53 @@ compute_sigma <- function(sigma_group, sigma_ref, index_mar){
 }
 
 
-#' Title
+
+#' Strategies
 #'
-#' @description
-#' TODO
+#' These functions are used to implement various reference based imputation
+#' strategies by combining a subjects own distribution with that of
+#' a reference distribution based upon which of their visits failed to meet
+#' the MAR assumption
+#'
+#' @param pars_group A list of parameters for the subject's group. See details.
+#'
+#' @param pars_ref A list of parameters for the subject's reference group. See details.
+#'
+#' @param index_mar A logical vector indicating which visits meet the MAR assumption
+#' for the subject. I.e. this identifies the observations that after a non-MAR
```

Comment:
without "that"

Reply:
I had actually missed out the word occur,  have replaced it with: 

> this identifies the observations that --occur-- after a non-MAR intercurrent event (ICE)